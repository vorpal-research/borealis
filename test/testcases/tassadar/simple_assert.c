
void borealis_assert(int);

int main() {
	borealis_assert(0);
}