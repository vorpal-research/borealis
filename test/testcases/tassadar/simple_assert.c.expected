[
"assertion failed: call void @borealis_assert(i32 0) #2, !dbg !15"
]
